decidim-censuses/app/jobs/decidim/censuses/remove_duplicates_job.rb

```diff
@@ -0,0 +1,20 @@
+module Decidim
+  module Censuses
+    class RemoveDuplicatesJob < ApplicationJob
+
+      queue_as :default
+
+      def perform
+        duplicated = Census.select(:id_document)
```

Comment:
Every time you are assigning a local variable, think if it's worth it to extract this to a method. It usually improves readability.

```ruby
def perform
   duplicated_census.each do |id_doc|
   end
end

private

def duplicated_census
Census.select(:id_document).blablabla
end
```